cookiecutter.json

```diff
@@ -18,7 +18,7 @@
     "use_read_the_docs": "y",
     "sphinx_theme": "astropy-boostrap",
     "initialize_git_repo": "y",
-    "astropy_helpers_version": "v2.0",
+    "astropy_helpers_version": "v2.0.2",
```

Comment:
we may want to branch this to be v3.0 or 2.0.4 depending whether the users package is python3 only or not.

Reply:
I am not sure how easy that would be to do. Could you open it as a different issue?